package.json

```
@@ -79,9 +79,9 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.0.0",
-    "@react-native-community/cli": "2.0.0-alpha.16",
-    "@react-native-community/cli-platform-ios": "2.0.0-alpha.15",
+    "@react-native-community/cli": "^2.0.0-alpha.19",
```

```suggestion
    "@react-native-community/cli": "2.0.0-alpha.19",
```
we need to pin to certain versions or `next` tag because there's a `2.0.0` release we did a while back by mistake

---

Good catch!

---

Commited, thank you!